site/labs/lists/2d-lists.rst

```diff
@@ -52,7 +52,7 @@ Given the following code:
 
     .. code-block:: python
 
-        twoD = [['a', 'b', 'c'],
+        two_D = [['a', 'b', 'c'],
```

Comment:
i left the upper case `D` here intentionally as D is for dimension

Reply:
may as well write `two_dimensional` to avoid any ambiguity